chart/templates/configmap.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if .Values.extraConfigmaps }}
```

Comment:
`{{- if .Values.extraConfigmaps }}` is not needed here and can be removed.
If the list is empty - no op.